### ISSUE TITLE
Apprise Implementation

### ISSUE THREAD

**Issue by ToHindahl** [sibalzer/impfbot#50]
Hello,
for several different and easier ways of receiving notifications we could consider implementing Apprise (Discord, Telegram, etc.)

https://pypi.org/project/apprise/

**Comment by sibalzer** [comment]
Wow thats a lot of services 😲. And looks quite easy to implement.

As a proposal:
I would like to keep the current config/structure of the alert system and config, to keep compatibility. We could add Apprise as an new notifier. E.g. like this:
```ini

[APPRISE]
enable=true
service_urls=discord://webhook_id/webhook_token,matrix://hostname

```

**Comment by ToHindahl** [comment]
I've finished the implementation. How can I release this now

**Comment by sibalzer** [comment]
1. Fork this repro
2. Push your changes
3. Make a pull requests to the Beta Branch

**Comment by sibalzer** [comment]
closed by #83
